src/utils/date-helpers.ts

```diff
@@ -23,8 +23,8 @@ export const getRegistrationDate = (date: Omit<RegistrationDate, 'type'> | undef
     return null;
   }
   const year = date.year.padStart(4, '0'); // Values are padded to handle (erronuous) dates like year=2 etc.
-  const month = (date?.month ?? '1').padStart(2, '0');
-  const day = (date?.day ?? '1').padStart(2, '0');
+  const month = date?.month ? date.month : '1'.padStart(2, '0');
+  const day = date?.day ? date.day : '1'.padStart(2, '0');
```

Comment:
må ikke dette wrappes i en parentes for at det alltid skal paddes? kan se ut som det bare er "1" som vil paddes nå 🤔 
```suggestion
  const month = (date?.month ? date.month : '1').padStart(2, '0');
  const day = (date?.day ? date.day : '1').padStart(2, '0');
```

Reply:
Godt spørsmål. Jeg antok den gjaldt for hele uttrykket, og det jeg testet funket, men jeg kan slenge på en parantes for å være sikker.